src/Sylius/Bundle/ApiBundle/Serializer/ProductAttributeValueNormalizer.php

```diff
@@ -49,16 +49,19 @@ public function normalize($object, $format = null, array $context = [])
 
         switch ($object->getType()) {
             case SelectAttributeType::TYPE:
+                Assert::isArray($data);
```

Comment:
Should we move `Assert` to `$this->normalizer->normalize($object, $format, $context);` line above `switch` case? 

Sorry, didn't catch this on the first time.

Reply:
I'm not sure about this! Currently, it will be a Breaking Change!